src/config/options.rs

```diff
@@ -458,6 +462,8 @@ impl From<Edition> for rustc_span::edition::Edition {
             Edition::Edition2018 => Self::Edition2018,
             Edition::Edition2021 => Self::Edition2021,
             Edition::Edition2024 => Self::Edition2024,
+            // TODO: update to 2027 after rustc change
+            Edition::Edition2027 => Self::Edition2024,
```

Comment:
@calebcartwright rustc_span::edition::Edition doesn't have Edition2027 yet, how should we solve this problem